roles/sfs/templates/alfresco-shared-fs.service

```diff
@@ -1,12 +1,14 @@
 [Unit]
 Description=Alfresco Shared File Store
-After=syslog.socket network.target
+After=syslog.socket network.target local-fs.target remote-fs.target
 
 [Service]
 Type=simple
 User={{ username }}
 ExecStart={{ binaries_folder }}/ats-shared-fs.sh
 WorkingDirectory={{ ats_home }}
+Restart=on-failure
+RestartSec=60
```

Comment:
like alfresco-tengine-aio.service, but we probably want to add an `exec` in `ats-shared-fs.sh` before the java command to replace the previous bash with java process

Reply:
I think we should move away from simple, or get away from wrapper scripts instead